cogs/helpers/_taskValidator.py

```diff
@@ -13,29 +13,29 @@ def validate(fileName):
 
             tasks = data["tasks"]
             days = {
-                "0": "Sun",
-                "1": "Mon",
-                "2": "Tue",
-                "3": "Wednes",
-                "4": "Thurs",
-                "5": "Fri",
-                "6": "Satur",
-                "*": "any ",
+                "00": "Sun",
+                "01": "Mon",
+                "02": "Tue",
+                "03": "Wednes",
+                "04": "Thurs",
+                "05": "Fri",
+                "06": "Satur",
+                "*": "any day",
```

Comment:
This one was fine as it was, it was just the month one
```suggestion
                "*": "any ",
```